airflow_dags/open_elections/load_by_state.py

```diff
@@ -0,0 +1,402 @@
+from datetime import datetime
+import os
+import pandas as pd
+from typing import List, Tuple, Any
+import re
+import argparse
+from doltpy.core import Dolt
+from doltpy.core.write import import_list
+from doltpy.etl.loaders import insert_unique_key
+import logging
+import importlib
+
+logger = logging.getLogger(__name__)
+
+ELECTIONS_TABLE_SCHEMA = '''
```

Comment:
Why is this defined here if not used? 

I like to separate schema definition from data loading, and use failure as the mode to tell me a data load didn't work because the schema changed. I've found trying to account for schema changes in a data load has a tendency to succeed and produce bad resulting data. I'd rather fail, have to update the job, and then retry.

It seems you follow the above pattern in this job. So why define the schema at all?

Reply:
Yes, that's fair. I will just create a file `ca.sql` with the schema definition.